### PR TITLE
Update cc3d model/cytokine sub model

### PR DESCRIPTION
**Pull request by JulianoGianlupi**
The cytokine field itself now has decay. It represents not dissolution but leakage to outside the system, metabolization. 

Immune cells have a dictionary that has the total amount of cytokine it has uptaken. The value in the dictionary now decays such that the equilibrium value gives a high probability of activation (80%).

Immune cells when active secrete cytokine and chemotax on cytokine. Activation is now based on a hill function; input is current total cytokine uptaken, .5 value of hill is the ec50 for cytokine sensibility (literature), coefficient is 2. Once activated immune cells will stay activated for a minimum amount of time (arbitrary parameter).

Cytokines are secreted by the infected cells and active immune cells. The maximum secretion rate for infected cells is 10x that of immune cells. Both secretions are controlled by hill functions. The infected cells hill takes the internal assembled viral load as input. The immune cells hill takes the value of cytokine they are currently exposed to.  

Added some utility functions: 
  - `get_assembled_viral_load_inside_cell` returns cell.dict['Uptake'] + cell.dict['Assembled']
  - `total_seen_field` returns the total concentration seen over the whole cell (can be an estimation or not)

### CODE REVIEW

**Review by kbreitha — APPROVED** [review]

File: CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Simulation/CoronavirusLib.py [hunk 1]
```diff
@@ -138,3 +138,7 @@ def reset_viral_replication_variables(cell):
     cell.dict['Secretion'] = 0
     for k in vr_cell_dict_to_sym.keys():
         cell.dict[k] = 0
+
+def get_assembled_viral_load_inside_cell(cell):
+    return cell.dict['Uptake'] + cell.dict['Assembled']
```
Comment (tjsego) [inline CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Simulation/CoronavirusLib.py]: Note to add time step multiplication

**Review by tjsego — APPROVED** [review]
Looks good.

**Review by kbreitha — COMMENTED** [review]
Great, thanks!

File: CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Simulation/CoronavirusLib.py (hunk 1 above)
Comment (JulianoGianlupi) [reply CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Simulation/CoronavirusLib.py]: done

**Review by joaponte — APPROVED** [review]

**Pull request merged**